README.md

```diff
@@ -11,7 +11,7 @@ Kickoff your store with this boilerplate. This starter ships with the main Fasts
 
 ## ⚠️ Before you start
 
-As of Dec, 22, 2021, this starter is still far from covering most basic cases found on VTEX. To summarise what we still do not support that is considered basic on the VTEX commerce platform, we prepared the list below. If the feature you want is listed, you can either wait for us to add support to the feature, or fork the repo and implement on your own. Note that, by forking the repo, you will miss new features and improvements we do in this repo and you will need a developer to backport the feature to your store. Finally, this list is a work in progress, so some features may be missing from both base.store starter and this list.
+As of Dec, 22, 2021, this starter is still far from covering most basic cases found on VTEX. To summarize what we still do not support that is considered basic on the VTEX commerce platform, we prepared the list below. If the feature you want is listed, you can either wait for us to add support to the feature, or fork the repo and implement on your own. Note that, by forking the repo, you will miss new features and improvements we do in this repo and you will need a developer to backport the feature to your store. Finally, this list is a work in progress, so some features may be missing from both base.store starter and this list.
```

Comment:
Are we aiming for British spelling or American spelling? 👀

Reply:
Not sure, @hellofanny. I opted for the American one since I believe to be more common in our writing. I think we'll still see more occurrences of "analyse, summarise" here and there, but it's less common for people to write "centre, colour, behaviour, licence... etc" intentionally.